ratis-grpc/src/main/java/org/apache/ratis/grpc/GrpcTlsConfig.java

```diff
@@ -98,26 +97,25 @@ public GrpcTlsConfig(PrivateKey privateKey, X509Certificate certChain,
 
   public GrpcTlsConfig(File privateKeyFile, File certChainFile,
```

Comment:
This method is not used anywhere, can we delete it?

Reply:
@captainzmc , thanks for reviewing this.  This method is used in Ozone.